docker-compose.yaml

```diff
@@ -0,0 +1,16 @@
+version: '3'
+services:
+  
+  db:
+    image: mongo:latest
+    ports: 
+        - 27017:27017
+    volumes:
+      - "dbdata:/var/lib/mongodb/data"
+      - "./db/seeds.js:/docker-entrypoint-initdb.d/seeds.js:ro"
+    environment: 
+      - MONGO_INITDB_ROOT_USERNAME=user
```

Comment:
Perhaps you could have created a .env file for your db details and then put it in .gitignore so it wasn't pushed up to github for security